tests/func/test_delta.py

```diff
@@ -96,6 +96,101 @@ def create_delta_dataset(ds_name):
     create_delta_dataset(ds_name)
 
 
+def test_delta_update_unsafe(test_session, tmp_dir, tmp_path):
+    catalog = test_session.catalog
+    default_namespace_name = catalog.metastore.default_namespace_name
+    default_project_name = catalog.metastore.default_project_name
+
+    starting_ds_name = "starting_ds"
+    merge_ds_name = "merge_ds"
+    dependency_ds_name = (
+        f"{default_namespace_name}.{default_project_name}.{starting_ds_name}"
+    )
+    dependency_ds_merge_name = (
+        f"{default_namespace_name}.{default_project_name}.{merge_ds_name}"
+    )
+    ds_name = "delta_ds"
+    ds_name = "delta_ds"
+
+    images = [
+        {"name": f"img{i}.jpg", "data": Image.new(mode="RGB", size=(64, 128))}
+        for i in range(1, 5)
+    ]
+
+    merge_images = [
+        {
+            "name": f"img{i}.jpg",
+            "data": Image.new(mode="RGB", size=(64, 128)),
+            "value": i,
+        }
+        for i in range(1, 5)
+    ]
```

Comment:
This feels more complicated than necessary for testing unsafe. Why Image is needed? Why not just File? You also probably don't need to create an actual image.

Another thought - why not using simple structure like read_values(num=[1, 2, 3, 4, 5], status="ready1", "ready2", "ready3", "ready4", "ready5") and run delta on it?